webprofile/11/_index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Jakarta EE Web Profile 11 (Under development)"
+date: 2023-07-17
+summary: "Release of the Jakarta EE 11 Web Profile"
+---
+The Jakarta EE Web Profile defines a profile of the Jakarta EE Platform specifically targeted at web applications.
+
+### New features, enhancements or additions
+<!-- List here -->
+* Support for Java Records 
+* Support for Virtual Threads
+
```

Comment:
Add 

```
* Jakarta Data 1.0
```

per https://dev.azure.com/jakarta-ee-azdo/jakarta-ee-azdo/_workitems/edit/171

Reply:
@edburns done the change